pkg/compactor/block_upload.go

```diff
@@ -303,10 +328,12 @@ func (c *MultitenantCompactor) UploadBlockFile(w http.ResponseWriter, r *http.Re
 	level.Debug(logger).Log("msg", "uploading block file to bucket", "destination", dst, "size", r.ContentLength)
 	reader := bodyReader{r: r}
 	if err := userBkt.Upload(ctx, dst, reader); err != nil {
-		level.Error(logger).Log("msg", "failed uploading block file to bucket", "operation", op, "destination", dst, "err", err)
 		// We don't know what caused the error; it could be the client's fault (e.g. killed
 		// connection), but internal server error is the safe choice here.
-		http.Error(w, "internal server error", http.StatusInternalServerError)
+		// If customer complains with "I got an internal server error" give them something we could correlate to our logs.
+		id := hexTimeNowNano()
```

Comment:
We don't need to create new `error_id` here, right?

Reply:
Right, I missed this one.